content/v1/resiliency/deployment/helm.md

```diff
@@ -194,6 +196,37 @@ podmon:
       - "--ignoreVolumelessPods=false"
 ```
 
+## PowerMax Specific Recommendations
```

Comment:
Hi @delldubey ,

The changes look good to me. I just have a small query: I am unsure if the changes should be in the file under the "v1" directory or in the "docs" directory.

Thanks!
<img width="216" alt="image" src="https://github.com/dell/csm-docs/assets/109620911/687d23ac-3b72-438c-b31c-50d871357314">